deploy/deploy.yaml

```diff
@@ -159,11 +161,18 @@ spec:
                       description: InProgressNodesCount reflects the number of nodes
                         that are in the process of kata installation
                       type: integer
+                    isInProgress:
+                      description: IsInProgress reflects if installation is still
```

Comment:
Just out of curiosity, why is the capitalisation consistently different between the name of the field (e.g. here `isInProgress`) and the name in the documentation (`IsInProgress`)? It's so consistent that it's clearly intentional, but now I won't sleep until I know why ;-)

Reply:
it is some convention, but tbh I just followed the example here :-)